smoderp2d/runoff.py

```diff
@@ -208,11 +217,16 @@ def __init__(self, provider):
             ma.masked_array(
                 np.zeros((GridGlobals.r, GridGlobals.c)), mask=GridGlobals.masks
             ),
-            True
+            Globals.isStream
```

Comment:
What is the purpose of this change? It changes results of the explicit solution. Does it have some purpose or was it created by mistake?

Reply:
Probably the residue of looking for a problem with saving the values in Streams. Corrected.

Reply:
Corrected.